Champions.ts

```diff
@@ -0,0 +1,35 @@
+import { Player } from "./models/Types";
+
+
+function findChampions(players: Player[]): Player[] {
+
+  if (players.length < 2) return players;
```

Comment:
this line is a bit of a pity, because the algo below works with length 0 and 1 as well, so no need to extract a particular case.

also, it makes is that the algo sometimes returns a new reference, and sometimes its own input, which can have side effects down the line